src/storage/src/source/healthcheck.rs

```diff
@@ -346,8 +346,8 @@ impl TryFrom<&str> for SourceStatus {
 
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub struct SourceStatusUpdate {
-    status: SourceStatus,
-    error: Option<String>,
+    pub status: SourceStatus,
```

Comment:
while we're here:  consider changing `SourceStatus` to mirror `SinkStatus` and have some variants hold an error `String` -- instead of a separate `Option<String>`

I think it'd be a good change but don't want to block this PR if you want to defer / not do